### DISCUSSION TITLE
stuck on pay store login, when i clickk login nothing happens

### DISCUSSION THREAD

**Discussion by richenberg** [LSPosed/MagiskOnWSALocal#169] [Q&A] [answered]
stuck on pay store login, when i click login nothing happens

![image](https://user-images.githubusercontent.com/44688332/194325735-5c219f45-0a92-4f6d-b776-fb198b0161dd.png)

**Comment by Howard20181** [comment, marked as answer]
It was fixed 20 days ago

**Comment by richenberg** [comment]
i changed version of WSA from 2208 to 2207 and fixed

**Reply by Howard20181** [reply]
In fact, 2208.40000.5.0 also works properly and can login to Google account.

**Comment by ccowan93** [comment]
I am also having this issue with version 2207.40000.8.0

<img width="640" alt="WsaClient_c7sSFRXRVk" src="https://user-images.githubusercontent.com/25260428/195696983-73a9a688-23f7-4d6e-8cbf-b714234c2145.png">

**Reply by Howard20181** [reply]
Please upgrade script.
